doc/api/buffer.md

```diff
@@ -1223,7 +1223,7 @@ added: v6.0.0
 -->
 
 * `value` {String | Buffer | Integer} What to search for
-* `byteOffset` {Integer} Where to begin searching in `buf` (not inclusive).
+* `byteOffset` {Integer} Where to begin searching in `buf`.
   **Default:** [`buf.length`]
```

Comment:
Default is actually `buf.length - 1`.

https://github.com/nodejs/node/blob/e21e12944c11f9d30e154d735f35b1f528cd7dc2/lib/buffer.js#L594